### PR TITLE
BEFH-BETAFPVF411.config

### PR DESCRIPTION
**Pull request by hongzhou-zeng**
BEFH-BETAFPVF411.config

### CODE REVIEW

File: configs/default/Add BEFH-BETAFPVF411.config [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
```
Comment (mikeller) [inline configs/default/Add BEFH-BETAFPVF411.config]: The name of the file is wrong now, please rename and remove the 'Add '.

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: I had removed the Add

Comment (mikeller) [reply configs/default/Add BEFH-BETAFPVF411.config]: Still there, try again: 
image

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: I had removed the Add again

Comment (mikeller) [reply configs/default/Add BEFH-BETAFPVF411.config]: image
Nope, still there.

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: image

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: Now is it right?

Comment (mikeller) [reply configs/default/Add BEFH-BETAFPVF411.config]: No. You need to change the file name, not the title of the pull request:

image

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: image

Comment (hongzhou-zeng) [reply configs/default/Add BEFH-BETAFPVF411.config]: I should hou to delete the same name's file?

**Pull request closed without merging**